packages/fuel-indexer-api-server/src/uses.rs

```diff
@@ -173,6 +173,38 @@ pub(crate) async fn stop_index(
     Err(ApiError::default())
 }
 
+pub(crate) async fn revert_index(
+    Path((namespace, identifier)): Path<(String, String)>,
+    Extension(tx): Extension<Option<Sender<ServiceRequest>>>,
+    Extension(pool): Extension<IndexerConnectionPool>,
+) -> ApiResult<axum::Json<Value>> {
+    let mut conn = pool.acquire().await?;
+
+    let _ = queries::start_transaction(&mut conn).await?;
+
+    if let Err(_e) =
+        queries::penultimate_index_id_for(&mut conn, &namespace, &identifier).await
```

Comment:
TIL [penultimate](https://www.merriam-webster.com/dictionary/penultimate). Love it.